### ISSUE TITLE
[RyuJIT/armel] CodeGenBringUpTests Assertion failed 

### ISSUE THREAD

**Issue by wateret** [dotnet/coreclr#11929]
A sub-issue for #11822 

Some tests fail with below assert failure.

```
               Assert failure(PID 17906 [0x000045f2], Thread: 17906 [0x45f2]): Assertion failed 'inArgMask & RBM_FLTARG_REGS' in 'BringUpTest:method_2I2D4S(int,int,double,double,struct,struct,struct,struct):int' (IL size 271)

                   File: /opt/code/github/hqueue/coreclr/src/jit/regalloc.cpp Line: 709
```

List of the tests fail with above message in `JIT/CodeGenBringUpTests`

```
FAILED   - [ 157]JIT/CodeGenBringUpTests/struct16args/struct16args.sh
```

**Comment by alpencolt** [comment]
There is the same issue on `JIT/CodeGenBringUpTests/FPVar/FPVar.exe` on softfp mode.
I will look to it.

**Comment by alpencolt** [comment]
Fixed by #12105 and #12005
